loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
Ok, but this list is composed of Issues and PRs, so there are not only issues closed.

So we should use something different to issues and bugs. What do you suggest?